Kros.Utils/src/Kros.Utils/Extensions/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Kros.Extensions
```

Comment:
I suggest namespace `System.Linq`. @satano, what do you think about it?

Reply:
Yes.